pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -1654,6 +1684,7 @@ func drWithLabels(lbls labels.Instance) *model.ConsolidatedDestRule {
 
 func TestConcurrentBuildLocalityLbEndpoints(t *testing.T) {
 	test.SetForTest(t, &features.CanonicalServiceForMeshExternalServiceEntry, true)
+	test.SetForTest(t, &features.EnableDualStack, true)
```

Comment:
If the feature flag has to be set for this to work, please add a separate test, instead of setting it in an existing test

Reply:
okay, done!